src/tests/shared_delegate_website_functions_test/shared_delegate_website_functions_test.c

```diff
@@ -59,12 +59,12 @@ int shared_delegate_website_functions_test(void)
   memset(result_test,0,sizeof(result_test));
   memcpy(result_test,"{\"public_address\":\"",19);
   memcpy(result_test+strlen(result_test),xcash_wallet_public_address,XCASH_WALLET_LENGTH);
-  memcpy(result_test+strlen(result_test),"\",\"total_vote_count\":\"2000000000\",\"IP_address\":\"127.0.0.1\",\"delegate_name\":\"delegate_name_1\",\"about\":\"about\",\"website\":\"website\",\"team\":\"team\",\"pool_mode\":\"false\",\"fee_structure\":\"0\",\"server_settings\":\"server settings\",\"block_verifier_score\":\"0\",\"online_status\":\"true\",\"block_verifier_total_rounds\":\"1\",\"block_verifier_online_total_rounds\":\"1\",\"block_verifier_online_percentage\":\"100\",\"block_producer_total_rounds\":\"2\",\"block_producer_block_heights\":\"|440875|440876\",\"public_key\":\"ad98cadb3b13229c78709876955247cbef40d5b15c4842be605b0e8b30c97a7a\"}",547);
+  memcpy(result_test+strlen(result_test),"\",\"total_vote_count\":\"2000000000\",\"IP_address\":\"127.0.0.1\",\"delegate_name\":\"delegate_name_1\",\"about\":\"about\",\"website\":\"website\",\"team\":\"team\",\"shared_delegate_status\":\"false\",\"delegate_fee\":\"0\",\"server_specs\":\"server settings\",\"block_verifier_score\":\"0\",\"online_status\":\"true\",\"block_verifier_total_rounds\":\"1\",\"block_verifier_online_total_rounds\":\"1\",\"block_verifier_online_percentage\":\"100\",\"block_producer_total_rounds\":\"2\",\"block_producer_block_heights\":\"|440875|440876\",\"public_key\":\"ad98cadb3b13229c78709876955247cbef40d5b15c4842be605b0e8b30c97a7a\"}",547);
```

Comment:
lengths

Reply:
Update with commits 5c5ead50a03c8e94d17a01a430ae9b713f40f122 and bf152cccaf69f79f48938e45b2af77e79a3a9c96